debug.xml

```diff
@@ -194,6 +194,10 @@
         SELECT * FROM groups;
     </entry>
 
+    <entry key='database.selectDefaultGroup'>
+        SELECT * FROM groups where name = 'The default group';
```

Comment:
I do prefer not needing that as well.  I didn't know the admin could see all devices.